packages/tslint-config/tslint.json

```diff
@@ -22,6 +22,7 @@
     "no-duplicate-variable": [true, "check-parameters"],
     "no-floating-promises": true,
     "no-invalid-template-strings": true,
+    "no-namespace": true,
```

Comment:
Becomes `@typescript-eslint/no-namespace` in the ESLint

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md